Xamarin.Forms.Platform.iOS/Cells/ViewCellRenderer.cs

```diff
@@ -84,6 +94,12 @@ public override void LayoutSubviews()
 				var contentFrame = ContentView.Frame;
 				var view = ViewCell.View;
 
+				if (Forms.IsiOS11OrNewer)
+				{
```

Comment:
This should really be up to the developer to decide. This will break every all alignments to content above/below listview when on ios11. This gap will also show the color of the Elements backgroundcolor that the listview is layed out on... Posted image and code in the forums for SR4

Reply:
please open a bug so we can investigate, i understand your point.

Reply:
@rmarinho @bentmar I can confirm that all of my ViewCells are shifted over on the left in iOS 11 with this release.  This occurs on all types of simulators.  Not sure why there would be left padding as a default.  This should only occur on landscape on a X to avoid the notch.

This is a breaking change